src/data-access/facades/repository-facade.ts

```diff
@@ -140,6 +142,14 @@ export class RepositoryFacade {
     )
   }
 
+  public static createIngestedMediaChangedListener(): DataChangedListener<Media> {
```

Comment:
You are listening on `Media`, so this should just be named `createMediaChangedListener`.

As I have mentioned before, the "ingested" entities we have are just for Rundown, Segment, Part and Piece so that we are able to "reset" them back to their original data. That is not something we need with Media.

Reply:
Renamed to the suggestion!